cloudflare.py

```diff
@@ -17,6 +17,7 @@
 ZONE = config('ZONE')
 AUTH_EMAIL = config('AUTH_EMAIL')
 AUTH_KEY = config('AUTH_KEY')
+TAGS = config('TAGS', default=["source:cloudflare"])
```

Comment:
Let's change this to 

```
 TAGS = config('TAGS', default="source:cloudflare,", cast=Csv) 
```
to enable us to declare multiple tag in the environment separated by comma